nodejs/week3/warmup/index.js

```diff
@@ -0,0 +1,54 @@
+const express = require("express");
+const app = express();
+const router = express.Router();
+app.get("/", (req, res) => res.send("nodejs week3 homework"));
+// Parse URL-encoded bodies (as sent by HTML forms)app.use(express.urlencoded({ extended: true }));// Parse JSON bodies (as sent by API clients)app.use(express.json());
```

Comment:
You actually need this to parse parameters from the body, it shouldn't be commented out:
```
app.use(
  express.urlencoded({
    extended: true
  })
)
```